linetools/isgm/utils.py

```diff
@@ -416,6 +434,26 @@ def table_from_complist(complist):
     assert len(key_order) == len(tab.keys())
     tab = tab[key_order]
 
+    # May need to add an HI component here as done in the method below
+    if NHI_obj is not None:
+        # Existing row in Table?
+        mt = np.where((tab['Z'] == 1) & (tab['ion']==1))[0]
+        if len(mt) == 1:
+            tab[mt[0]]['logN'] = NHI_obj.NHI
+            tab[mt[0]]['sig_logN'] = np.mean(NHI_obj.sig_NHI) # Allow for two values
```

Comment:
how about having a column for `sig_logN_plus` and another for `sig_logN_minus`

Reply:
Worth discussing, but then I need to treat sig_logN and
sig_N different all the time from other attributes.  Lots
of extra coding..